### ISSUE TITLE
WPTexturize is mangling curly quotes in the post title when shared to twitter

### ISSUE THREAD

**Issue by jb510** [jaredatch/Shared-Counts#71]
## Plugin version
1.2.0

## Current Behavior
Using the post title
'But We've Always...'

and sharing to twitter the tweet comes up as
&#8216;But We&#8217;ve Always&#8230;&#8217; 

## Expected Behavior
For the post title to be based to twitter without encoded characters

## Possible Solution
This is happening because of wptexturize running on get_the_title() so my relatively hacky quick fix was to temporarily disable wptexturize around get_the_title().

class-shared-count-front.php around line 470

```
// Temporarily disable wptexturize.
$wptexturize = remove_filter( 'the_title', 'wptexturize' );

$link['title'] = wp_strip_all_tags( get_the_title( $id ) );

// Re-enabled wptexturize.
if ( $wptexturize ) {
	add_filter( 'the_title', 'wptexturize' );
}
```

I feel like there is a maybe a better way.

## Steps to Reproduce (for bugs)
1. Create post with title 'But We've Always...'
2. Note that those entitles get converted to curly quotes and an ellipse when the title it displayed in a theme
3. Click the tweet button
4. Note what is displayed is not &#8216;But We&#8217;ve Always&#8230;&#8217; 

## Screenshots (if appropriate)
![2018-12-04 at 11 08 pm](https://user-images.githubusercontent.com/801241/49491407-a1f72680-f819-11e8-89d5-8779db961e41.png)

**Comment by jb510** [comment]
@jaredatch Did this actually get patched? I'm not seeing a commit to fix it...  and as said in my issue, I didn't put in a PR for my "fix" because it was pretty hacky...

**Comment by jaredatch** [comment]
It was already fixed last year in GH, but hasn't made it out because we haven't pushed a release.

I updated the changelog so it gets noted for the next release.

Original commit is https://github.com/jaredatch/Shared-Counts/commit/47c7a8c9255ea96761aeb3c7ff5970f5e4c2f0e5#diff-49da578f70f197f8c6eb94a367e94b88

I tested the `develop` branch and everything is working good (now) :)

**Comment by jb510** [comment]
Gotcha... and confirmed the version of SharedCounts I forked for this fix was 1.2 and used rawurlencode.

Thanks.  We'll revert from our forked version to this one when it's released.
